x-pack/platform/plugins/shared/alerting/server/application/rule/methods/bulk_edit_params/schemas/bulk_edit_rule_params_option_schemas.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+
+const exceptionListSchema = schema.object({
+  id: schema.string(),
+  list_id: schema.string(),
+  type: schema.oneOf([
+    schema.literal('detection'),
+    schema.literal('rule_default'),
+    schema.literal('endpoint'),
+    schema.literal('endpoint_trusted_apps'),
+    schema.literal('endpoint_events'),
+    schema.literal('endpoint_host_isolation_exceptions'),
+    schema.literal('endpoint_blocklists'),
+  ]),
+  namespace_type: schema.oneOf([schema.literal('single'), schema.literal('agnostic')]),
+});
+
+const bulkEditExceptionListSchema = schema.object({
+  operation: schema.literal('set'),
+  field: schema.literal('exceptionsList'),
+  value: schema.arrayOf(exceptionListSchema),
+});
+
+export const bulkEditParamsOperationSchema = schema.oneOf([bulkEditExceptionListSchema]);
```

Comment:
In the future, we should be able to add schemas to support updating the investigation guide fields

Reply:
I am wondering if we should allow any params to be updated and move the responsibility to the function's consumers. This way, the framework can be agnostic of the rule types' params.

Reply:
Updated in https://github.com/elastic/kibana/pull/227891/commits/1d393e13e29ed7c1f7b4e14398ab9548d09931b0